osu.Game/Overlays/OnScreenDisplay.cs

```diff
@@ -114,11 +115,12 @@ public OnScreenDisplay()
         }
 
         [BackgroundDependencyLoader]
-        private void load(FrameworkConfigManager frameworkConfig)
+        private void load(FrameworkConfigManager frameworkConfig, OsuConfigManager osuConfig)
         {
             trackSetting(frameworkConfig.GetBindable<FrameSync>(FrameworkSetting.FrameSync), v => display(v, "Frame Limiter", v.GetDescription(), "Ctrl+F7"));
             trackSetting(frameworkConfig.GetBindable<string>(FrameworkSetting.AudioDevice), v => display(v, "Audio Device", string.IsNullOrEmpty(v) ? "Default" : v, v));
             trackSetting(frameworkConfig.GetBindable<bool>(FrameworkSetting.ShowLogOverlay), v => display(v, "Debug Logs", v ? "visible" : "hidden", "Ctrl+F10"));
+            trackSetting(osuConfig.GetBindable<bool>(OsuSetting.MouseDisableButtons), v => display(!v, "Mouse Buttons", v ? "Disabled" : "Enabled", "F10"));

```

Comment:
Can we rename the config setting to "GameplayMouseButtons" with default value true? Which would also clean up the whole `v => !v, v` stuff going on in here